Makefile

```diff
@@ -1110,6 +1152,48 @@ LDFLAGS     = -lm \
               -Wl,--no-wchar-size-warning \
               -T$(LD_SCRIPT)
 
+#SITL compile
+ifeq ($(TARGET),$(filter $(TARGET),$(SITL_TARGETS)))
+CFLAGS      = $(ARCH_FLAGS) \
+              $(addprefix -D,$(OPTIONS)) \
+              $(addprefix -I,$(INCLUDE_DIRS)) \
+              $(DEBUG_FLAGS) \
+              -std=gnu99 \
+              -Wall -Wextra -Wunsafe-loop-optimizations -Wdouble-promotion \
+              -ffunction-sections \
+              -fdata-sections \
+              -pedantic \
+              $(DEVICE_FLAGS) \
+              -DUSE_STDPERIPH_DRIVER \
+              -D$(TARGET) \
+              $(TARGET_FLAGS) \
+              -D'__FORKNAME__="$(FORKNAME)"' \
+              -D'__TARGET__="$(TARGET)"' \
+              -D'__REVISION__="$(REVISION)"' \
+              -save-temps=obj \
+              -MMD -MP
+
+ASFLAGS     = $(ARCH_FLAGS) \
+              -x assembler-with-cpp \
+              $(addprefix -I,$(INCLUDE_DIRS)) \
+              -MMD -MP
+
+LDFLAGS     = \
+              -lm \
+              -lpthread \
+              -lc \
+              -lrt \
+              $(ARCH_FLAGS) \
+              $(LTO_FLAGS) \
+              $(DEBUG_FLAGS) \
+              -static \
+              -static-libgcc \
+              -Wl,-gc-sections,-Map,$(TARGET_MAP) \
+              -Wl,-L$(LINKER_DIR) \
+              -Wl,--cref \
+              -T'src/main/target/SITL/parameter_group.ld'
```

Comment:
`-nostartfiles`,`--specs=nano.specs`, `-lnosys` in `LDFLAGS` must remove.
and `-lpthread`, `-lc`, `-lrt` are in order.
so, maybe can remove `CFLAGS` and `ASFLAGS`,
but remove `LDFLAGS` need some more change.